app/design/frontend/rwd/default/template/email/catalog/product/list.phtml

```diff
@@ -84,13 +84,11 @@ $_params = $this->escapeHtml(json_encode(['form_key' => $this->getFormKey()]));
                                 <div class="product-secondary">
                                     <?php if(!$_product->canConfigure() && $_product->isSaleable()): ?>
                                         <p class="action">
+                                            <?php $_cartUrl = $this->getAddToCartUrlCustom($_product, [], false); ?>
                                             <button type="button"
                                                     title="<?php echo $this->quoteEscape($this->__('Add to Cart')) ?>"
                                                     class="button btn-cart"
-                                                    onclick="customFormSubmit(
-                                                            '<?php echo $this->getAddToCartUrlCustom($_product, [], false) ?>',
-                                                            '<?php echo $_params ?>',
-                                                            'post')">
+                                                    onclick="customFormSubmit('<?php echo $_cartUrl ?>', '<?php echo $_params ?>', 'post')">
```

Comment:
isn't this code exactly the same as before?

Reply:
@fballiano I looked at the history of this file and didn't find this code.